app/controllers/graphql_controller.rb

```diff
@@ -30,4 +30,31 @@ def ensure_hash(ambiguous_param)
       raise ArgumentError, "Unexpected parameter: #{ambiguous_param}"
     end
   end
+
+  def current_user
+    if doorkeeper_token && !doorkeeper_token.expired?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/d439434a286bd76fead4ad367fe89e1a1cb8da1b/app/controllers/graphql_controller.rb#L35"></span>Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>